### ISSUE TITLE
Nested/composite key and required set to true causes `Expression has changed after it was checked error`

### ISSUE THREAD

**Issue by gkamperis** [ngx-formly/ngx-formly#3380]
**Description**
At some point soon after 

```
"@ngx-formly/bootstrap": "5.10.28",
"@ngx-formly/core": "5.10.28",
```

The below bug started to happen.
Was avoiding upgrade hoping it will be spotted and fixed but seems not to be the case.

It took a bit of head-banging to narrow down but:

if there is a nested (foo.bar) `key` specified in the config for a field *and* the `required: true` the tests fail with the error described below.

Works ok for 
* nested key + `required: false`
* simple key + `required: true`

Fails for
* nested key + `required: true`

**Minimal Reproduction**
std ng 13 project found here: 
https://github.com/gkamperis/formly-bug

just run tests
`npm run test --watch`

```
AppComponent BUG FAILED
        Error: NG0100: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value for 'ng-valid': 'true'. Current value: 'false'.. Find more at https://angular.io/errors/NG0100
        error properties: Object({ code: -100 })
        Error: NG0100: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value for 'ng-valid': 'true'. Current value: 'false'.. Find more at https://angular.io/errors/NG0100
            at throwErrorIfNoChangesMode (node_modules/@angular/core/fesm2020/core.mjs:6759:11)
            at bindingUpdated (node_modules/@angular/core/fesm2020/core.mjs:12761:17)
            at checkStylingProperty (node_modules/@angular/core/fesm2020/core.mjs:16517:32)
            at ɵɵclassProp (node_modules/@angular/core/fesm2020/core.mjs:16425:5)
            at hostBindingFn (node_modules/@angular/forms/fesm2020/forms.mjs:1389:1)
            at processHostBindingOpCodes (node_modules/@angular/core/fesm2020/core.mjs:9277:17)
            at refreshView (node_modules/@angular/core/fesm2020/core.mjs:9556:9)
            at refreshComponent (node_modules/@angular/core/fesm2020/core.mjs:10707:13)
            at refreshChildComponents (node_modules/@angular/core/fesm2020/core.mjs:9306:9)
            at refreshView (node_modules/@angular/core/fesm2020/core.mjs:9560:13)
```

https://github.com/gkamperis/formly-bug/blob/master/src/app/app.component.spec.ts

**Your Environment**

"@ngx-formly/bootstrap": "^5.12.5",
"@ngx-formly/core": "^5.12.5"

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v5.12.6](https://github.com/ngx-formly/ngx-formly/releases/tag/v5.12.6) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!

**Comment by osmankiraz** [comment]
I have same error.But I am using v6.0.2. Could you help me to resolve ?
